README.md

```diff
@@ -786,7 +786,7 @@ case of the `indexvm`, the custom rule support is used to set the cost for
 adding anything to state (which is a very `hypervm-specific` value).
 
 You can view what the import `Action` associated with the above examples looks like
```

Comment:
The text here still says "import". I think a slightly larger change is required.

Reply:
Oupsie sorry, I was understanding it like "used" term.
It should be better now if CamelCase Action name is accepted